specifications.yml

```diff
@@ -5,6 +5,11 @@ info:
   description: |
     The Fimento Open API can be used by Fimento's partners in order to provide financial products to their end customers within their own application/website.
 
+    <a href="./postman_collection.json" download> [Download Postman collection] </a>
+
+    To use the collection, update the `UNAME` and `PWORD` in the root folder of the collection to the credentials received from Fimento.
+
+    To automate the collection, set the authorization token to be `{{TOKEN}}`.
```

Comment:
I think we could have a whole section "Postman" for these description actually.
We could have this text and also have a text for the alternative to download the openAPI version and import it as it is in postman but specify that there will be less automation than the one we built